src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+  width: 96px;
+  height: 16px;
```

Comment:
no needs to hardcode height and width for stars block

Reply:
If I change these values, my stars look different from the layout